libcontainerd/container_windows.go

```diff
@@ -42,13 +42,18 @@ func (ctr *container) start() error {
 	// until the container is done with the servicing execution.
 	logrus.Debugln("Starting container ", ctr.containerID)
 	if err = ctr.hcsContainer.Start(); err != nil {
-		logrus.Errorf("Failed to start compute system: %s", err)
+		logrus.Errorf("Failed to start container: %s", err)
+		if err := ctr.terminate(); err != nil {
+			logrus.Errorf("Failed to cleanup after a failed Start. Ignoring this. %s", err)
```

Comment:
Oh! Didn't see @LK4D4's comment. 
For me, it was mainly the text including "ignoring", and the type being "error" that confused me. Perhaps the error message should be slightly rephrased then, but I may be nit-picking